aws/resource_aws_backup_region_settings_test.go

```diff
@@ -0,0 +1,123 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/backup"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAwsBackupRegionSettings_basic(t *testing.T) {
+	var settings backup.DescribeRegionSettingsOutput
+
+	rName := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "aws_backup_region_settings.test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSBackup(t) },
```

Comment:
```suggestion
		PreCheck:     func() {
			testAccPreCheck(t)
			testAccPartitionHasServicePreCheck(fsx.EndpointsID, t)
			testAccPreCheckAWSBackup(t)
		},
```

This should fix the issue

Reply:
That will skip the resource, the API is support but the setting can be set are different in gov and commercial regions. is this the behaviour we want?